<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MailNotifyPageClass</class>
 <widget class="QWidget" name="MailNotifyPageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>268</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="twtMails"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbtNewMail">
       <property name="text">
        <string>Send mail</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbtIncoming">
       <property name="text">
        <string>Incoming</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
